packages/react-vapor/src/components/title/tests/Title.spec.tsx

```diff
@@ -92,4 +95,37 @@ describe('<Title/>', () => {
             expect(titleComponent.find(LinkSvg).prop('linkClasses')).toContain('inline-doc-link');
         });
     });
+
+    it('renders a tooltip if the title is truncated', async () => {
+        // mock the element DOM properties
+        const originalScrollWidth = Object.getOwnPropertyDescriptor(HTMLElement.prototype, 'offsetHeight');
+        const originalOffsetWidth = Object.getOwnPropertyDescriptor(HTMLElement.prototype, 'offsetWidth');
+
+        Object.defineProperty(HTMLElement.prototype, 'scrollWidth', {

```

Comment:
You could not spy on those I guess?

Reply:
I tried to spyon the ref. But I didn't try on the `HTMLElement.prototype` though